File: Python/Material.py
class Material:
    def __init__(self, jenis="", nama=""):
        self.jenis_bahan = jenis
        self.nama_bahan = nama

    def set_jenis_material(self, jenis):
        self.jenis_bahan = jenis

    def get_jenis_material(self):
        return self.jenis_bahan

    def set_nama_material(self, nama):
        self.nama_bahan = nama

    def get_nama_material(self):
        return self.nama_bahan

    def add_data_material(self, jenis, nama):
        self.jenis_bahan = jenis
        self.nama_bahan = nama

File: javanies/Material.java
public class Material {
    private String jenisBahan;
    private String namaBahan;

    public Material() {}

    public Material(String jenis, String nama) {
        this.jenisBahan = jenis;
        this.namaBahan = nama;
    }

    public void setJenisMaterial(String jenisBahan) {
        this.jenisBahan = jenisBahan;
    }

    public String getJenisMaterial() {
        return jenisBahan;
    }

    public void setNamaMaterial(String namaBahan) {
        this.namaBahan = namaBahan;
    }

    public String getNamaMaterial() {
        return namaBahan;
    }

    public void addDataMaterial(String jenisBahan, String namaBahan) {
        this.jenisBahan = jenisBahan;
        this.namaBahan = namaBahan;
    }
}
